packages/ckeditor5-emoji/src/insertemojicommand.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module emoji/insertemojicommand
+ */
+
+import type { Range } from 'ckeditor5/src/engine.js';
+import { Command } from 'ckeditor5/src/core.js';
+
+export default class InsertEmojiCommand extends Command {
```

Comment:
I think we should not create this `InsertEmojiCommand`, because it does not do what it says. It just inserts plain text, and it could be missleading for the end user. Besides in this code it is already used in a wrong way to insert empty space.